src/utils/fetch.ts

```diff
@@ -1,16 +1,28 @@
-﻿/**
- * @module saxo/utils/fetch
- * @ignore
- */
-
-import log from '../log';
-
-// -- Local variables section --
+﻿import log from '../log';
+
+type HttpMethod =
+    | 'GET'
+    | 'HEAD'
+    | 'POST'
+    | 'PUT'
+    | 'DELETE'
+    | 'CONNECT'
+    | 'OPTIONS'
+    | 'TRACE'
+    | 'PATCH';

```

Comment:
I will extract this type together with the successful response type in a separate PR since it will be used in other places.